playbooks/install_stack.yaml

```diff
@@ -393,8 +398,7 @@
     script: files/export-dcn.sh
 
   - name: Reboot if SR-IOV is enabled (to apply kernel changes)
-    when:
-      - sriov_interface is defined or dpdk_kernel_args is defined
+    when: false
```

Comment:
Hmm, interesting. Did you really intend to change this line?

Reply:
yeah I'm debugging something. the PR is in draft for now.